streaming/src/test/java/org/apache/spark/streaming/LocalJavaStreamingContext.java

```diff
@@ -22,6 +22,8 @@
 import org.junit.After;
 import org.junit.Before;
 
+@SuppressWarnings( "deprecation" )
```

Comment:
hmm.. why need to suppress `deprecation` here?

Reply:
we don't want to remove these dstream tests, but also don't want the deprecation message to pollute the build logs.

Reply:
Technically all DStream tests should have produced deprecation warning message, otherwise we don't do the job very well. That said, dealing with only one file is not that critical. We have to either suppress deprecation as many as possible, but if we can technically do a couple of tests, doing nothing is OK.

Reply:
Looks like doing this in Scala side is sensitive to Scala version. For Scala 2.12, it depends on the bugfix version.

https://www.scala-lang.org/2021/01/12/configuring-and-suppressing-warnings.html

Let's leave the tests to produce deprecation warning messages, until we figure out the way to address both Scala and Java.

Reply:
Reverted.